### DISCUSSION TITLE
do u eat pizza

### DISCUSSION THREAD

**Discussion by chitang233** [KimmyXYC/clash-rules#4] [Q&A] [answered]
我不敢苟同你的观点，我觉得吃披萨应该加95号汽油，并且应该加到试管底部1/3的位置，这样是为了拉格朗日点能够在 爱因斯 坦质能方程中找到属于自己的北回归线，如果不小心加错加了92号汽油，那就不好办了，首先要找到老爹让他念出戊戌变法，防止发动机和神经中枢对撞产生等离子火花和伽马射线，按照孔子的说法，我们还要把进化论里的化学方程式配平，这样才能让海尔兄弟放过舒克和贝塔

**Comment by pchpub** [comment]
合理

**Comment by KimmyXYC** [comment, marked as answer]
感觉你们好有趣啊，不像我，不仅连句话都搭不上，还要被当成破坏氛围的傻狗，我现实生活中自闭没朋友，哪怕是在网上也受尽冷眼，每次组织了半天的语言都如鲠在喉，最后还是默默删掉了看你们互动，你有说有笑的样子不知道为什么在我眼里这么刺眼，融入不了群体的我，躲在屏幕后面默默哭出来了，所以今天是肯德基疯狂星期四有好心人请我吃吗？
